[esx_addons]/esx_lscustom/server/main.lua

```diff
@@ -33,6 +34,20 @@ AddEventHandler('esx_lscustom:buyMod', function(price)
 	end
 end)
 
+RegisterNetEvent('esx_lscustom:changeVehicleData')
+AddEventHandler('esx_lscustom:changeVehicleData', function(type, data)
+	local source = source
+	if type == "add" then
+		VehicleData[source] = {plate = data.plate, props = data.props}
+	elseif type == "remove" then
+		for index, value in ipairs(VehicleData) do
```

Comment:
You can replace the loop with :

```lua
if VehicleData[source] then VehicleData[source] = nil end
```

Reply:
HI, @mahanmoulaei
Maybe he/she custom multi cars
Again, a specific data must be deleted